README.md

```diff
@@ -2642,6 +2643,58 @@ Produces a random number between the inclusive lower and upper bounds. If only o
 
 **[⬆ back to top](#quick-links)**
 
+### _.clamp
+
+Clamps number within the inclusive lower and upper bounds.
+
+```js
+// Lodash
+_.clamp(-10, -5, 5);
+// => -5
+ 
+_.clamp(10, -5, 5);
+// => 5
+
+_.clamp(10, -5);
+// => -5
+
+_.clamp(10, 99);
+// => 10
+
+// Native
+const clamp = (number, boundOne, boundTwo) => {
+  if (!boundTwo) {
```

Comment:
- The second argument is optional according to the docs: https://lodash.com/docs/4.17.15#clamp
-  With only two arguments, the function treats the first as the target number and the second as the upper bound
- With all three, the arguments are the target number, the lower bound, and then the upper bound
- So, if there are only two arguments, the number only has to be `clamped` against the upper bound